mm2src/mm2_main/src/lp_swap/maker_swap.rs

```diff
@@ -2235,7 +2237,7 @@ pub async fn maker_swap_trade_preimage(
 
     let preimage_value = TradePreimageValue::Exact(volume.to_decimal());
     let base_coin_fee = base_coin
-        .get_sender_trade_fee(preimage_value, FeeApproxStage::TradePreimage)
+        .get_sender_trade_fee(preimage_value, FeeApproxStage::TradePreimage, NO_REFUND_FEE)
```

Comment:
Why refund fee is not included in maker trade preimage?

Reply:
This is supposed to be one of the fixes for too high fees. I consulted with QA and they said that users may refuse from starting the swap when they see too big total fee for it.
So this change to create the trade preimage tx with no refund fee was to decrease the total fee shown to the user. I checked the GUI code and believe they use the trade preimage tx as a source to show swap fee to the user - so I think now user will fee only for the payment tx.
At the same time, the max_taker_vol rpc still includes refund fee, so the GUI can check it and will not allow to run a swap if user cannot have all needed funds for refund.

I guess this is debatable and easy to turn off/on in the code, I decided to add this for experimenting how it may work in the GUI.

Reply:
> I guess this is debatable and easy to turn off/on in the code

We can turn it off/on using the trade preimage request instead. We can have this the default behaviour with the option to include refund fee if this parameter is true in the request. No need to fix this now but maybe in another PR after this is merged or when/if this feature was requested by QA/GUI.

Reply:
yes, a param for the GUI to include or not refund fee would be better

Reply:
I even think, it is better to add refund fee to swap data as a dedicated field (and also return it in the trade preimage response), because max swap volume is also checked by mm2 internally during the swap and better to include refund fee there

Reply:
added an item into todo in the PR description